recipe/meta.yaml

```diff
@@ -34,20 +36,56 @@ requirements:
     - python-xxhash
     - multiprocess
     - importlib-metadata  # [py<38]
+    # fsspec[http]
     - fsspec >=2021.05.0
+    - aiohttp
     - huggingface_hub >=0.7.0
     - packaging
-    - responses <0.19
-    - cookiecutter # to populate metric template. Required by CLI: https://github.com/huggingface/evaluate/blob/v0.3.0/src/evaluate/commands/evaluate_cli.py#L6
```

Comment:
We should keep `cookiecutter` in `run` because it's required for the main entry_point of the package, see https://github.com/huggingface/evaluate/blob/v0.4.3/setup.py#L139 and https://github.com/huggingface/evaluate/blob/5310084f128a86f6969cc3fe1aaf0573d6859dbc/src/evaluate/commands/evaluate_cli.py#L6. In other words if we ship the package's entry_point, we ship `cookiecutter`.

I think the maintainers wrongly decided to have it in `extras_require` https://github.com/huggingface/evaluate/blob/v0.4.3/setup.py#L141 and https://github.com/huggingface/evaluate/blob/v0.4.3/setup.py#L122